allensdk/model/biophysical/_schemas.py

```diff
@@ -0,0 +1,8 @@
+import argparse
+
+# Create the parser
+sim_parser = argparse.ArgumentParser(description='arguments to be passed for biophysical simualtions')
```

Comment:
small thing, but this is the description that shows up when someone invokes the executable, so it should describe the executable and not the arguments thereof